docs/src/user-guide/getting-started.md

```diff
@@ -23,9 +23,40 @@ You can install and configure ESLint using this command:
 
 ```shell
 npm init @eslint/config
+
+
+# or
+
+yarn init @eslint/config
+
+# or
+
+pnpm create @eslint/config
```

Comment:
https://github.com/eslint/eslint/issues/16050#issuecomment-1165831664

We talked about it earlier and decided to not add all package managers

> I'd prefer not to start adding every package manager into the docs. I wonder if we can just now say npm init @eslint/config rather than instructing people to manually install ESLint?

Reply:
That's correct. We aren't including every option. When they run `npm init @eslint/config`, it will ask if they want to use yarn or pnpm. There's no need to show multiple options here.